www/artists/jcowin/works.html

```diff
@@ -14,259 +14,259 @@
   <div class="mySlides fade">
     <div class="numbertext">1 / 53</div>
     <img src="img/000.jpg" style="width:100%">
-    <div class="text"></div>
+    <div class="text">1st image</div>
```

Comment:
...This isn't alt text. Alt text is the stuff in the alt within the img tag, such as <img src="img/000.jpg" alt="1st image">. It's not just text in a div immediately following the image.

Reply:
I will create a new pull request fixing this.

Reply:
An entirely new pull request isn't necessary. If you make the changes to this branch, this pull request will automatically update to include them.